astropy/coordinates/earth.py

```diff
@@ -83,8 +83,7 @@ def _get_json_result(url, err_str, use_google):
         #   http://stackoverflow.com/questions/2712524/handling-urllib2s-timeout-python
         if isinstance(e.reason, socket.timeout):
             raise NameResolveError(err_str.format(msg="connection timed out")) from e
-        else:
```

Comment:
Same

Reply:
Here, I'd avoid the ugliness with
```
msg = (
    "connection timed out"
    if isinstance(e.reason, socket.timeout)
    else e.reason
)
raise NameResolveError(err_str.format(msg=msg)) from e
```

Should add that I really don't know why we bother with prettifying exception messages in so much detail...